docs/release_notes.rst

```diff
@@ -10,6 +10,7 @@ New Data Coverage
 ^^^^^^^^^^^^^^^^^
 * Add EIA860M data through December 2023 :issue:`3313`, :pr:`3367`.
 * Add 2023 Q4 of CEMS data. See :issue:`3315`, :pr:`3379`.
+* Add EIA923 monthly data through November 2023 :issue:`3314`, :pr:`3398`.
```

Comment:
Does RMI understand that the 923 data in this update only runs through November, 2023? Were they expecting data through the end of Q4 for all the datasets?